plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -54,39 +54,39 @@ module.exports = ({containerConfig, appConfig}) => {
       </Row>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {(fields.resources.cpus != null && fields.resources.cpus > 0) && (
```

Comment:
Hey @mlunoe , this actually follows the [react best practices for in-line conditional rendering](https://facebook.github.io/react/docs/conditional-rendering.html#inline-if-with-logical-ampamp-operator). And I wouldn't say that's really "logic" there. This is more like conditional rendering.

A more complicate logic would definitely need a separate function, or even perhaps a local component.

What do you think /cc @orlandohohmeier ?